common/src/models/project.rs

```diff
@@ -85,8 +85,9 @@ pub struct ProjectUpdateRequest {
     pub team_id: Option<String>,
     /// Transfer away from current team
     pub remove_from_team: Option<bool>,
-    /// Change compute tier
-    pub compute_tier: Option<ComputeTier>,
+    /// Project runtime configuration
+    #[cfg_attr(feature = "utoipa", schema(value_type = String, format = "json"))]
+    pub config: Option<Box<serde_json::value::RawValue>>,
```

Comment:
I would use `Option<serde_json::Value>` and `serde_json::from_value()` on the other side. Saves the need of an extra feature.

Reply:
Unfortunately this serializes it as a string, so you need to deserialize it twice on the other side.

Reply:
You sure? https://play.rust-lang.org/?version=stable&mode=debug&edition=2024&gist=fe5bd881adf39aefa656d176b5803988

Reply:
Seems to work now... I guess you can't have it as `serde_json::Value` in Clap's arguments.